src/PrestaShopBundle/Resources/config/routing/admin/sell/orders/_orders.yml

```diff
@@ -12,7 +12,7 @@ _carts:
 
 _orders:
   resource: "orders.yml"
-  prefix: /orders/
```

Comment:
@jolelievre This change does not seem to be needed for this PR 🤔 ? Also it's a big change, can you explain ?